sled-agent/src/bootstrap/agent.rs

```diff
@@ -965,40 +994,33 @@ impl Agent {
     }
 }
 
-// We intentionally DO NOT derive `Debug` or `Serialize`; both provide avenues
+// We intentionally DO NOT derive `Debug``; it provides avenues
```

Comment:
I'd maybe replace this with something like

```suggestion
// TODO-correctness: remove / rework trust quorum share storage
```

since we're not really caring about storage of these shares (which are effectively always `None` anyway).

Reply:
Done